Automattic/jetpack
# Docs: Update the Connection Manager namespace, a minor typo

#### Changes proposed in this Pull Request:
_Jetpack Connection Docs._
Class `Automattic\Jetpack\Manager` was renamed to `Automattic\Jetpack\Connection\Manager`.
I'm updating the documentation according to these changes. 

_Life Cycle of a PR documentation_
A minor typo fixed in the `pull-request.md`.

#### Testing instructions:
Review the files and confirm they render correctly:
* https://github.com/Automattic/jetpack/blob/fix/docs-updates/docs/pull-request.md
* https://github.com/Automattic/jetpack/blob/fix/docs-updates/packages/connection/docs/register-site.md
* https://github.com/Automattic/jetpack/blob/fix/docs-updates/packages/connection/docs/authorize-user.md

#### Proposed changelog entry for your changes:
n/a
